sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Insert-disbursement-blocked-message.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO
+  sims.notification_messages(id, description, template_id)
+VALUES
+  (
+    16,
+    'Disbursement withheld',
+    'bf53ce0d-c59c-4d03-badb-e0cab04ac582'
+  ),
+  (
+    17,
+    'Disbursement blocked ministry notification',
```

Comment:
Minor. I would ask to follow same term here. `Disbursement withheld ministry notification`